doc/changes/latest.inc

```diff
@@ -109,6 +109,8 @@ Enhancements
 
 - The :meth:`mne.Evoked.get_data` method now has a ``units`` parameter (:gh:`9578` by `Stefan Appelhoff`_)
 
+- Add :func:`mne.get_montage_rois` to find the regions of interest in a Freesurfer atlas anatomical segmentation for an intracranial electrode montage and :func:`mne.viz.plot_channel_labels_circle` to plot them (:gh:`9545` by `Alex Rockhill`_)
```

Comment:
Why tend to calls ROIs labels everywhere in mne / freesurfer world. So here it would be volume labels?

Get_montage_volume_labels or get_montage_vol_labels ?

Reply:
+ 1 for `get_montage_volume_labels`